src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.tsx

```diff
@@ -108,7 +135,9 @@ const CreateFromInstanceType: FC<CreateFromInstanceTypeProps> = ({
                 sectionKey={INSTANCE_TYPES_SECTIONS.VM_DETAILS}
                 sectionState={sectionState}
               >
-                <VMDetailsSection />
+                <VMDetailsSection
+                  instanceTypesAndPreferencesData={instanceTypesAndPreferencesData}
+                />
               </SectionListItem>
             </List>
           </Card>
```

Comment:
u changed to pass the store as a prop instead of consuming it?

Reply:
instanceTypesAndPreferencesData and bootableVolumesData was taken out of the store, and now the store consists only of props for the VM config